CheckDrive.Api/CheckDrive.Domain/Exceptions/InvalidLoginAttemptException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CheckDrive.Domain.Exceptions;
+
+public class InvalidLoginAttemptException : Exception
+{
+    public InvalidLoginAttemptException() : base() { }
+    public InvalidLoginAttemptException(string message) : base(message) { }
+    public InvalidLoginAttemptException(string message, Exception inner) : base(message, inner) { }
```

Comment:
Why u use this constructor this, Other manually exceptions deos not have this constructor type